aggregator.go

```diff
@@ -105,6 +105,30 @@ func closeStorage(storage *storage.DBStorage) {
 	}
 }
 
+func prepareDBMigrations(dbStorage *storage.DBStorage) int {
+	// This is only used by some unit tests.
+	if autoMigrate {
```

Comment:
do we need to use global var there?

Reply:
@tisnik The only alternative I could think of was adding a field into the config file, but that felt like even more effort.

Reply:
I meant that `autoMigrate` might be parameter passed into the function.

Reply:
I don't want to stop your work and this one is very minor thing